test/data/certificate_testdata.js

```diff
@@ -0,0 +1,71 @@
+module.exports = [
```

Comment:
Copied this from https://github.com/HorizenOfficial/bitcore-lib-zen/blob/mo/zendoo/test/data_tx/tx-sidechain-certificate.json